test/extended/util/annotate/rules.go

```diff
@@ -131,6 +131,7 @@ var (
 		},
 
 		"[Feature:Networking-IPv4]": {
+			`\[sig-network\]\[Feature:Router\] when FIPS is disabled the HAProxy router should serve routes when configured with a 1024-bit RSA key \[Suite:openshift/conformance/parallel\]`,
```

Comment:
Not sure if it’s needed but I would take off the suite annotation at the end.  And you have to run make generate or similar

Reply:
yes, `./hack/update-generated.sh` -- thx